docker/Dockerfile

```diff
@@ -61,6 +61,13 @@ RUN curl -sL https://github.com/mongodb/mongo-c-driver/releases/download/1.13.0/
     && cmake --build mongo-cxx-build --target install \
     && rm -rf mongo-cxx-*
 
+# libCrypto requires openssl-1.1.x.
+RUN curl -sL https://www.openssl.org/source/openssl-1.1.1.tar.gz | tar -zx -C . \
```

Comment:
should we also add the instruction to our readme doc

Reply:
Yes, we should, for those who are running Ubuntu 16.04, they'll have to do this.